examples/distributed_ray_train/train.py

```diff
@@ -1,81 +1,154 @@
 import argparse
```

Comment:
Is this from https://github.com/ray-project/ray/blob/master/python/ray/train/examples/pytorch/torch_fashion_mnist_example.py?

If so, I'm wondering if we should we change the YAML to directly wget/git clone the example from ray repo so we don't have to maintain a copy?